### ISSUE TITLE
[Bug] Unknown error code 10

### ISSUE THREAD

**Issue by Vaskivskyi** [Vaskivskyi/ha-asusrouter#490]
> I just upgraded my router form RT AC86U to RT AX86U. Asusrouter intergration used to work just fine. Now with new router add asusrouter integration ends with unknown error.
> 
> This error originated from a custom integration.
> 
> Logger: custom_components.asusrouter.config_flow
> Source: custom_components/asusrouter/config_flow.py:584
> Integration: AsusRouter (documentation, issues)
> First occurred: 09:44:43 (1 occurrences)
> Last logged: 09:44:43
> 
> Unknown error of type '<class 'homeassistant.exceptions.ConfigEntryNotReady'>' during connection to 192.168.1.1: ('Unknown error code 10', None)
> 
> The official AsusWRT integration works with new router.
> 
> Home assistant setup is the same as Whitetigerswt, exept running HASSOS 9.3 on RPI4.

_Originally posted by @jullimullins in https://github.com/Vaskivskyi/ha-asusrouter/issues/414#issuecomment-1369475379_

**Comment by Vaskivskyi** [comment]
Hello, @jullimullins,

Please, let me know whether you have managed to set up AsusRouter with your new RT-AX86U or do you still experience an `Unknown error code 10` error as quoted from your post above.

If so, please, answer the following:
- Can you login into the Web UI of the device from the browser?
- When setting up the configuration, do you select to use SSL connection?
- Do you select the connection port manually?

**Comment by Vaskivskyi** [comment]
The reason for the `error code 10` found - this is a captcha request from the device

The integration will implement the correct message

**Comment by edwardtfn** [comment]
I got this same error today and the integration isn't loading:

![IMG_6125](https://user-images.githubusercontent.com/94725493/228153317-30fbeb78-639d-425d-a9b8-9d5d810b2633.jpeg)


```log
This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/asusrouter/bridge.py:173 
Integration: AsusRouter (documentation, issues) 
First occurred: 03:53:23 (1 occurrences) 
Last logged: 03:53:23

Error setting up entry AiMesh - Router for asusrouter
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 383, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/asusrouter/__init__.py", line 26, in async_setup_entry
    await router.setup()
  File "/config/custom_components/asusrouter/router.py", line 632, in setup
    await self.bridge.async_connect()
  File "/config/custom_components/asusrouter/bridge.py", line 173, in async_connect
    await self.api.async_connect()
  File "/usr/local/lib/python3.10/site-packages/asusrouter/asusrouter.py", line 253, in async_connect
    raise ex
  File "/usr/local/lib/python3.10/site-packages/asusrouter/asusrouter.py", line 251, in async_connect
    await self.connection.async_connect()
  File "/usr/local/lib/python3.10/site-packages/asusrouter/connection.py", line 298, in async_connect
    response = await self.async_request(
  File "/usr/local/lib/python3.10/site-packages/asusrouter/connection.py", line 210, in async_request
    raise AsusRouterResponseError(
asusrouter.error.AsusRouterResponseError: ('Unknown error code `10`, please report it', None)
```

Please let me know if you want more info. I'm not in a rush, so I can run some test if that would be useful for you.

**Comment by Vaskivskyi** [comment]
Hello, @edwardtfn, sorry for the late reply. Unfortunately, this error cannot be processed anyhow from the integration point of view, except of just notifying the user properly. The latest will be available with the next integration version

**Comment by Vaskivskyi** [comment]
The error origin is the request from the device to enter a captcha. This might happen with too strict device rules or if a wrong password was used several times lately. The only 'fix' will be to login to the device Web UI from the same IP as the integration tries, properly in a browser - then the captcha request will reset. Or to just wait for some time. Or remove captcha completely in the device settings

**Comment by salvq** [comment]
Usually I restart HTTP daemon in asus router and this helps in 99%. However not sure what is triggering and why HTTP got stacked.

`service restart_httpd`

**Comment by Vaskivskyi** [comment]
The issue is fixed in `0.23.0b0` and later. The error code is reported correctly by the integration
